src/resolvers/teams.js

```diff
@@ -1,7 +1,26 @@
-const { sequelize } = require('../models');
-const models = require('../models');
 const retry = require('retry-as-promised');
 const { Sequelize } = require('sequelize');
+const { sequelize } = require('../models');
+const models = require('../models');
+const { sendEmail } = require('../services/sendEmail');
+
+// TO DO: Edit inivitation email subject, html, and include team name (along with link)
+const createTeamInviteMessage = teamId => {
+  // convert teamId to teamName
+  return {
+    from: 'hongyichen@uwblueprint.org',
+    subject: `Invitation to Join Building Up`,
+    html: 'You have been invited to join <strong>Building Up</strong>. Please join using this link: ',
+  };
+};
+
+const sendTeamInvites = (emails, teamId) => {
+  const message = createTeamInviteMessage(teamId);
+  Promise.all(emails).then(email => {
+    const invitationEmail = { to: { email }, ...message };
+    sendEmail(invitationEmail);
```

Comment:
`Promise.all` should be called on an Array of `Promise`. It's quite confusing, but it would be something like this:

```
const sendTeamInvites = async (emails, teamId) => {
  const message = createTeamInviteMessage(teamId);
  return Promise.all(
    emails.map(email => {
      const invitationEmail = { to: { email }, ...message };
      return sendEmail(invitationEmail);
    }),
  );
};
```

Notes:
* understanding async operations in javascript is easily the most confusing part of the language (in particular, promises). So don't worry if you're having a hard time understanding/figuring out the syntax
* It's especially hard to see what's going on without TypeScript, lol (I complain about not having TS quite often)
* `Promise.all` is especially tricky, I hope my code sample is correct lol

Reply:
Thanks for explaining this! This makes a lot of sense now